### ISSUE TITLE
JSR-303 validator failed to initialize

### ISSUE THREAD

**Issue by simonjenga** [jcabi/jcabi-dynamodb-maven-plugin#73]
```
C:\javaworkspace\javaworkspace\jcabi-dynamodb-maven-plugin>mvn clean package
.
.
.
Results :

Tests run: 4, Failures: 0, Errors: 0, Skipped: 0

[INFO]
[INFO] --- jcabi-maven-plugin:0.13.2:versionalize (jcabi-versionalize-packages) @ jcabi-dynamodb-maven-plugin ---
[INFO] Versionalizing C:\javaworkspace\jcabi-dynamodb-maven-plugin\target\classes directory
[INFO] org.hibernate.validator.internal.util.Version: HV000001: Hibernate Validator 5.1.2.Final
[ERROR] JSR-303 validator failed to initialize: Unable to instantiate Configuration. (see http://aspects.jcabi.com/jsr-303.html)
[INFO] File C:\javaworkspace\jcabi-dynamodb-maven-plugin\target\classes\com\jcabi\dynamodb\core\com-jcabi-jcabi-dynamodb-maven-plugin-maven-plugin.txt added
[INFO] File C:\javaworkspace\jcabi-dynamodb-maven-plugin\target\classes\com\jcabi\dynamodb\maven\plugin\com-jcabi-jcabi-dynamodb-maven-plugin-maven-plugin.txt added
[INFO]
[INFO] --- maven-jar-plugin:2.6:jar (default-jar) @ jcabi-dynamodb-maven-plugin ---
[INFO] Building jar: C:\javaworkspace\jcabi-dynamodb-maven-plugin\target\jcabi-dynamodb-maven-plugin-1.0-SNAPSHOT.jar
[INFO]
[INFO] --- maven-plugin-plugin:3.4:addPluginArtifactMetadata (default-addPluginArtifactMetadata) @ jcabi-dynamodb-maven-plugin ---
[INFO]
[INFO] --- maven-assembly-plugin:2.6:single (assemble-all) @ jcabi-dynamodb-maven-plugin ---
[INFO] Building jar: C:\javaworkspace\jcabi-dynamodb-maven-plugin\target\jcabi-dynamodb-maven-plugin-1.0-SNAPSHOT-jar-with-dependencies.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@simonjenga[/z](https://www.0crat.com/u/simonjenga) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3RUBL5H9) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by yegor256** [comment]
@simonjenga what's wrong? I see the build is clean.

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-dynamodb-maven-plugin#73` is not assigned, can't get performer

**Comment by simonjenga** [comment]
@yegor256 There is an issue with the Build: 
```[ERROR] JSR-303 validator failed to initialize: Unable to instantiate Configuration. (see http://aspects.jcabi.com/jsr-303.html)```

It should be fixed like it was done here: https://github.com/jcabi/jcabi-http/issues/187

**Comment by simonjenga** [comment]
@yegor256 This is where the problem is.
To fix it, we Bump parent to ```1.24``` by upgrading to ```jcabi 1.24```
The PR has been submitted at #75
